src/main/java/com/park/easyrecruit/servlet/Login.java

```diff
@@ -0,0 +1,35 @@
+package com.park.easyrecruit.servlet;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ * @author M Radu
+ */
+@WebServlet(name = "Login", urlPatterns = {"/Login"})
```

Comment:
lowercase login maybe?

Reply:
I followed the convention used by Alex Dorobantiu in 10b-D.Reorganize URLs.